docker/build_scripts/build.sh

```diff
@@ -152,7 +144,7 @@ ln -s $PY36_BIN/auditwheel /usr/local/bin/auditwheel
 # final image
 yum -y erase wireless-tools gtk2 libX11 hicolor-icon-theme \
     avahi freetype bitstream-vera-fonts \
-    ${PYTHON_COMPILE_DEPS}  > /dev/null 2>&1
+    ${PYTHON_COMPILE_DEPS}  #> /dev/null 2>&1
```

Comment:
The redirection is there because at some point Travis start freaking out at our logs being too long. Not sure if we want to comment it out or not...

Reply:
Ah, yeah, in fact it looks like the x86-64 build is failing because we need to hide the glibc build logs. (Did *you* know that Travis will kill any job that generates too much logs?)

Reply:
To be fair, 4 MB is a _lot_ of logs.  Seems fair of Travis to assume some test fell into a loop.

What do you think about creating a new base image just for the patched glibc?  I was reluctant to do it at first because I was worried introducing another intermediary would make it harder to reproduce the image.  At the same time, we already rely on `centos:6.9` staying available in Docker Hub, so maybe depending on `centos-no-vsyscall` instead isn't so bad?

Also, it looks like Travis includes [a Docker that's new enough](https://travis-ci.org/pypa/manylinux/jobs/335881849#L36) to support [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/).  I'll make the x86_64 Dockerfile build  glibc in its own stage so the final image is a little smaller.

Reply:
I'll also mute `rpmbuild` for now - we can always move to another base image later.